### PR TITLE
Add Server API documentation

### PR DESCRIPTION
**Pull request by peyerluk**

### CODE REVIEW

**Comment by peyerluk** [comment]
Fixes #6

**Comment by peyerluk** [comment]
@gabrielhase @marcbachmann please review if everything is up to date.

File: server/editing_api_authentication.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# API Authentication
+### `POST /authenticate` Logging In
+
+#### Parameters
+
+| Property      | Type    | Default  | Description
+| ------------- | ------- | -------- | -------------
+| `username`    | string  | -        | **required**, utf8 encoded username, normally your email address
+| `password`    | string  | -        | **required**, utf8 encoded password
+| `expires`     | integer | 7200     | Access token expiration time in milliseconds
```
Comment (marcbachmann) [inline server/editing_api_authentication.md]: We've removed that expires param

File: server/deployment.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+```bash
+heroku config:set aws__secret_key="your-secret"
+heroku config:set aws__access_key="your-key"
+heroku config:set aws__buckets__images="livingdocs-images"
+```
+
+If you want to copy settings from other heroku instances you can use this heroku plugin: https://github.com/ddollar/heroku-config
+
+### Example Configuration
+(Those are not real passwords. so don't try to use them.)
```
Comment (marcbachmann) [inline server/deployment.md]: I hope that's really true ;)

File: server/integration_bridge.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#### authorize(access_token, callback)
+```
+  access_token: Access token string
+  callback: must be called with one of the following options
+    * callback(err) when an error occurs
+    * callback(null, {user: userObject, credentials: Object)
+    * callback() When unauthenticated
+```
+#### find({id}, callback)
```
Comment (marcbachmann) [inline server/integration_bridge.md]: I guess we can remove that file

**Comment by marcbachmann** [comment]
Do you like to delete the old gh wiki?

**Comment by gabrielhase** [comment]
> Do you like to delete the old gh wiki?

Lets wait a little with that. We should go through it and make sure first that we took everything over. Then we should delete it.

File: server/editing_api_authentication.md (hunk 1 above)
Comment (peyerluk) [inline server/editing_api_authentication.md]: Do you want to update this before I merge?

**Pull request merged**